tree/treeplayer/inc/ROOT/TDFOperations.hxx

```diff
@@ -33,15 +34,32 @@ using namespace Internal::TDFTraitsUtils;
 using Count_t = unsigned long;
 using Hist_t = ::TH1F;
 
+template<typename F>
+class ForeachSlotOperation {
```

Comment:
I am in favor of removing the redundancy.

I think it's worth mentioning, though, that `TDataFrameInterface` has a method called `ActionName` for each one of the classes `ActionNameOperation`, so just dropping that suffix might generate confusion for the reader. Would it not matter, since we seem to always refer to operations with their fully qualified name?

Just a bit of brain-storming on the naming: operations are, in fact, helpers of `TDataFrameAction`. They encapsulate the logic of each action. They are the de facto action executors (but the term executor is already used elsewhere). We could go as far as moving the `Operations` classes inside the `TDataFrameAction` class scope: they are not supposed to be used elsewhere.

I don't see an obviously better choice. What do you think? @dpiparo ?